python/lsst/l1dbproto/l1dbschema.py

```diff
@@ -262,11 +309,25 @@ def getAfwSchema(self, table_name, columns=None):
             else:
                 units = column.unit or ""
                 # some units in schema are not recognized by afw but we do not care
-                key = schema.addField(afw_col,
-                                      type=self._afw_type_map_reverse[column.type],
-                                      doc=column.description or "",
-                                      units=units,
-                                      parse_strict="silent")
+                if self._afw_type_map_reverse[column.type] == 'String':
+                    key = schema.addField(afw_col,
+                                          type=self._afw_type_map_reverse[column.type],
+                                          doc=column.description or "",
+                                          units=units,
+                                          parse_strict="silent",
+                                          size=10)
+                elif afw_col == 'coord_ra' or afw_col == 'coord_dec':
```

Comment:
I'm not happy about this, hardcoding column names at this level does not feel right. Can we instead use schema properties to do that? If we want all angles to be Angles in afw.table I would suggest to replace this line with:

    elif units == "deg":

Alternatively we can look at "ucd" property if we want more specific transformations.

But in general we need to think about more general type conversion model sometime later, database schema contain other angle-related combination of units like "deg^2", I'm not sure what should we do for those when we convert between afw and database.

Reply:
I've changed the test to what you have suggested.

Yeah, for now the code that would compute covariances and produce deg^2 units doesn't exist in the stack yet. I think we'll be needing it soon both for the association work and other locations. I think at that point we'll decide on what object type we store a "deg^2" unit in.